### PR TITLE
move railtie to proper namespace

### PR DESCRIPTION
**Pull request by chetan**
moved the Railtie into the GlobalIdentification module

### CODE REVIEW

**Comment by rafaelfranca** [comment]
We should move the file too.

File: lib/active_model/railtie.rb [hunk 1]
```diff
@@ -2,15 +2,17 @@
 require 'active_model/global_identification'
 
 module ActiveModel
-  # = Active Model GlobalID Railtie
-  class Railtie < Rails::Railtie # :nodoc:
-    initializer "active_model.globalid" do
-      config.after_initialize do |app|
-        ActiveModel::SignedGlobalID.verifier = app.message_verifier(:signed_global_ids)
-      end
+  module GlobalIdentification
```
Comment (rafaelfranca) [inline lib/active_model/railtie.rb]: Maybe GlobalID to match the gem name.

**Comment by sj26** [comment]
The file name is more important. The railtie will reopen the class nicely and is where the initializer is targeted to be included. `active_model/railtie` is a collision which prevents a rails app from booting at all.

Don't want to step on pull requests, but I've popped the adjustment on my fork's master for Gemfile use:

https://github.com/sj26/activemodel-globalid

**Comment by chetan** [comment]
@sj26 is spot on. I've updated the PR

**Comment by jeremy** [comment]
Fixed @ ef6f7a1d330b42618a913ffd1db0e3608df26613. Thanks @chetan !

**Pull request closed without merging**